src/t8_schemes/t8_default/t8_default_common/t8_default_common.hxx

```diff
@@ -74,60 +221,37 @@ class t8_default_scheme_common_c: public t8_eclass_scheme_c {
    * Note that this number is >= 1, since we count the element itself as a sibling.
    */
   int
```

Comment:
Please add a comment that pyramids have their own implementation of this function.